src/Components/Buttons/SortByLength.tsx

```diff
@@ -0,0 +1,18 @@
+import { SortingProps } from '../../types/buttons/SortingProps';
+import { SortType } from '../../Enums/SortType';
+import cn from 'classnames';
+import React from 'react';
+
+export const SortByLength: React.FC<SortingProps> = ({ sortBy, setSortBy }) => {
+  return (
+    <button
+      type="button"
+      className={cn('button is-success', {
+        'is-light': sortBy !== SortType.LENGTH,
+      })}
+      onClick={() => setSortBy(SortType.LENGTH)}
+    >
+      Sort by length
+    </button>
```

Comment:
[UX] - The button does not indicate if it is the active sorting method. Consider adding a visual indication to mark the button as active when `sortBy` is `SortType.LENGTH`. This could be achieved by changing the button's style or text when it is active.